app/views/layouts/application.html.erb

```diff
@@ -1,12 +1,14 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SamplePp</title>
+    <title> <%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
Em tìm hiểu i18n trong rails, rồi vận dụng vào project này, thay toàn bộ những text cứng trong code thành các biến i18n nhé